### PR TITLE
Testing, see https://github.com/linkml/linkml/issues/529

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

**Comment by cmungall** [comment]
redundant with #191

**Comment by hrshdhgd** [comment]
I'll delete my PR. We can merge this one.

File: setup.cfg [hunk 1]
```diff
@@ -52,14 +52,13 @@ install_requires =
...
     jsonasobj==1.3.1
     jsonasobj2>=1.0.4
-    linkml-runtime==1.1.10
+    linkml-runtime>=1.1.12
     networkx
     numpy
     pandas
     pandasql
     pyyaml
     rdflib==5.0.0
```
Comment (cthoyt) [inline setup.cfg:61]: Is upgrading to RDFlib 6 coming soon?

Comment (hrshdhgd) [reply setup.cfg:61]: Hasn't been on the cards for now. I could look into it if it's an easy transition.

**Comment by cmungall** [comment]
Yes, it's supported. rdflib5 support will be dropped in the 1.2 series of
linkml.

**Comment by cthoyt** [comment]
so if both are supported, then maybe it's a bit too restrictive to pin to rdflib 5 in sssom-py. @hrshdhgd do you want to make a PR to see if unpinning that breaks anything?

**Comment by hrshdhgd** [comment]
Doing it right now.

**Comment by hrshdhgd** [comment]
Done! It passes GH Actions checks. Should I merge directly or wait for either of you to approve?

**Comment by cthoyt** [comment]
:shipit: just ship it

**Pull request merged**